.travis.yml

```diff
@@ -60,7 +76,15 @@ before_script:
     - sed -i "s/yourusernamehere/travis/" wp-tests-config.php
     - sed -i "s/yourpasswordhere//" wp-tests-config.php
     - cd "/tmp/wordpress/src/wp-content/plugins/$PLUGIN_SLUG"
+    - |
+        if [[ ${TRAVIS_PHP_VERSION:0:2} == "7." ]]; then
+          composer global require "phpunit/phpunit=5.7.*"
+        elif [[ ${TRAVIS_PHP_VERSION:0:3} != "5.2" ]]; then
```

Comment:
Is this really needed? Under what circumstances the travis runs in PHP 5.2 environment now, when it was deprecated? I might be missing something, of course :)

Reply:
I added this part because PHPUnit 5 is working starting with PHP 5.6 and for PHP 5.4 and 5.5  the PHPUnit 4 is needed (ref: https://phpunit.de/). The actual error message that appears when not doing this change is:
Package phpunit/phpunit at version 5.7.* has a PHP requirement incompatible with your PHP version (5.4.45)

Reply:
Thanks for the additional details. I perhaps should had been more clear.

The conditional says that the

```
composer global require "phpunit/phpunit=4.8.*"
```

line should be triggered always, when the code runs outside PHP 7.x unless it's PHP 5.2. But, according to the configuration of used PHP versions, there should be no case when the code runs under PHP 5.2. Thus, the conditional could be simple `else`, imho.

Reply:
You are right about the else part. When the PHP version is other than 7.x, the phpunit version that can be used is 4.8.*. I'll fo this change in code.